framework/mesh/io/mesh_io.h

```diff
@@ -19,6 +19,8 @@ class MeshIO
   FromEnsightGold(const UnpartitionedMesh::Options& options);
   static std::shared_ptr<UnpartitionedMesh> FromOBJ(const UnpartitionedMesh::Options& options);
   static std::shared_ptr<UnpartitionedMesh> FromGmsh(const UnpartitionedMesh::Options& options);
+  static std::shared_ptr<UnpartitionedMesh> FromGmshV41(const UnpartitionedMesh::Options& options);
+  static std::shared_ptr<UnpartitionedMesh> FromGmshV22(const UnpartitionedMesh::Options& options);
```

Comment:
If the user facing API is `FromGmsh`, then `FromGmshV41` and `FromGmshV22` should be protected or private. Unless you want users to be able to call those two, which it does not look like it is the case...

Reply:
Fixed.